src/data/endpoints/managed.yaml

```diff
@@ -89,21 +89,34 @@ endpoints:
             curl -H "Authorization: Bearer $TOKEN" \
                 -X DELETE \
                 https://$api_root/$version/managed/contact/$contact_id
-
-  #/managed/history:
-  # group: History
-  #  type: resource
-  #  description: >
-  #    A list of the issues created from monitoring your Services.
-  #  methods:
-  #    GET:
-  #      response: ManagedHistory
-  #      description: >
-  #        Returns your managed service issue history list.
-  #      examples:
-  #        curl: |
-  #          curl -H "Authorization: Bearer $TOKEN" \
-  #              https://$api_root/$version/managed/history
+  /managed/issues:
+    group: Issues
+    authenticated: true
+    description: >
+      A list of the issues created from monitoring your Services.
+    methods:
+      GET:
+        response: ManagedIssue
+        description: >
+          Returns your managed service issue history list.
```

Comment:
I would remove the word "history" from here - this is a collection of your managed issues